src/main/resources/carpet.mixins.json

```diff
@@ -208,10 +208,10 @@
     "CaveScalerMixin_scarpetMixin",
 
     "BlockEntity_movableBEMixin",
-    "PistonBlock_movableTEMixin",
-    "PistonBlockEntity_movableTEMixin",
```

Comment:
Yeah, please don't rename them and leave them for subsequent PR where you can use the actual mojang class names as well.